scripts/imgtool/main.py

```diff
@@ -356,10 +379,32 @@ def sign(key, public_key_format, align, version, pad_sig, header_size,
         else:
             custom_tlvs[tag] = value.encode('utf-8')
 
+    #allow signature calculated externally
```

Comment:
Minor nit, there should be a space after the '#', the Allow capitalized, and a period at the end. See [PEP-8](https://peps.python.org/pep-0008/#comments).

Reply:
fixed